src/ui/components/Library/Team/View/NewTestRuns/TestRunsPage.tsx

```diff
@@ -0,0 +1,264 @@
+import { Suspense, useContext } from "react";
```

Comment:
This file might benefit from refactoring out some of the components into separate files. There's a lot going on here. Moving spec details and related out would be a good start, I think